modules/users/server/controllers/users/users.authentication.server.controller.js

```diff
@@ -116,7 +116,7 @@ exports.oauthCallback = function (strategy) {
           return res.redirect('/authentication/signin');
         }
 
-        return res.redirect(redirectURL || sessionRedirectURL || '/');
```

Comment:
I get @OneOfTheWorld point, the thing is, ```oauthCallback``` is actually called before ```saveOAuthUserProfile``` so even if we do:
```
user.save(function (err) {
  req.session.redirect_to = '/settings/accounts';
  return done(err, user);
});
```
the redirection will still not be applied.